docs/running-workflows/launching-workflows/index.md

```diff
@@ -0,0 +1,125 @@
+# Launch workflows from external source
+
+If your analysis workflows are hosted on external web sources such as GitHub, GitLab, Zenodo,
+it is possible to launch your workflows directly from the REANA web interface's `/launch` web page.
+You can also include a badge in your repository that will allow launching the workflow with a single click.
+
+For example, to launch a [RooFit demo workflow](https://github.com/reanahub/reana-demo-root6-roofit) from GitHub on REANA instance at CERN, you can click on the badge below:
+
+[![Launch RooFit workflow from GitHub](https://www.reana.io/static/img/badges/launch-on-reana-at-cern.svg)](https://reana.cern.ch/launch?url=https%3A%2F%2Fgithub.com%2Freanahub%2Freana-demo-root6-roofit)
+
+## Launcher arguments
+
+To create your own launcher URL, you will need to provide some information to REANA instance on how to launch the workflow.
+This takes a form of various arguments that can be passed in URL.
+The complete launch URL looks like this:
+
+```text
+https://<REANA_SERVER_URL>/launch?url=<WORKFLOW_SOURCE_URL>&name=<WORKFLOW_NAME>&specification=<WORKFLOW_SPEC_PATH>&parameters=<WORKFLOW_PARAMETERS>
+```
+
+Where:
+
+- `REANA_SERVER_URL` is an address of the REANA instance that will be used to launch the workflow;
+- `url`, `name`, etc. are arguments that provide information to launch a workflow.
+
+For example, the above badge for RooFit demo workflow is using this URL:
+
+```text
+https://reana.cern.ch/launch?url=https://github.com/reanahub/reana-demo-root6-roofit?parameters={"events": 10000}
+```
+
+Below you can find a list of all possible arguments, their description, and examples.
+
+### url (required)
+
+Below you can find a list of supported external workflow sources that can be used in `url` parameter.
+
+#### GitHub
+
+You can use GitHub repositories to launch a workflow. Below are examples of supported URL formats:
```

Comment:
You can use GitHub or GitLab repositories to launch a workflow from there, for example: